backend/src/main/java/com/thoughtworks/aceleradora/service/quickestimates/QuickEstimateCalculator.java

```diff
@@ -0,0 +1,37 @@
+package com.thoughtworks.aceleradora.service.quickestimates;
+
+import java.math.BigDecimal;
+
+import static com.thoughtworks.aceleradora.service.quickestimates.FloorAccess.ELEVATOR;
+import static com.thoughtworks.aceleradora.service.quickestimates.FloorAccess.GROUND_FLOOR;
+import static com.thoughtworks.aceleradora.service.quickestimates.Material.PLASTER;
+
+public class QuickEstimateCalculator {
+
+    public BigDecimal calculateEstimate(BagEstimateParameters parameters) {
+
+        if (parameters.getMaterial() == PLASTER) {
```

Comment:
Eu to viajando na maionese aqui, mas fiquei me perguntando se não conseguimos representar estas regras em forma de dados ao invés de condições. Algo nessa linha:

```java
   static final List<PriceCriteria> CRITERIA = asList(
            criteria(PLASTER, "200.00", GROUND_FLOOR, ELEVATOR),
            criteria(WOOD, "140.00", GROUND_FLOOR, ELEVATOR)
    );

    static class PriceCriteria {
        Material material;
        Set<FloorAccess> access;
        BigDecimal price;

        public PriceCriteria(Material material, Set<FloorAccess> access, BigDecimal price) {
            this.material = material;
            this.access = access;
            this.price = price;
        }

        static PriceCriteria criteria(Material material, String price, FloorAccess... access) {
            return new PriceCriteria(
                    material,
                    Arrays.stream(access).collect(Collectors.toSet()),
                    new BigDecimal(price)
            );
        }
    }
```